### PR TITLE
Update selecting directory.  Cleanup / refactor document tests.

### PR DESCRIPTION
**Pull request by toaster33**
Please review.  @vbudhram

### CODE REVIEW

**Comment by vbudhram** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -42,6 +42,6 @@
     "super-request": "0.0.8",
     "supertest": "^0.13.0",
     "supertest-as-promised": "^1.0.0",
-    "valid-url": "^1.0.9"
+    "validator": "^3.39.0"
```
Comment (DavidTPate) [inline package.json]: Instead of this, I think it would be best to use a fully compliant validator such as [isemail](https://www.npmjs.com/package/isemail) this way we are able to better handle edge cases other than the standard `something@somewhere.tld`.

File: bin/lrs-test.js [hunk 2]
```diff
@@ -6,10 +6,12 @@
 (function (process, require, program, exit, packageJson, Q, Joi, fs, path, Mocha) {
     'use strict';
 
+    var DIRECTORY = 'v1_0_2';
```
Comment (DavidTPate) [inline bin/lrs-test.js]: I see we have this value in environment variables also. Should we be pulling it from there here?
